src/content/projects/yordle.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: "/projects/yordle"
+date: "2022-06-27"
+title: "Yordle"
+preview: "../../images/Projects/Yordle/preview.png"
+client: "Yordevs"
+description: "After the rise in popularity of 'Wordle', we created a clone of the game where every answer was in some way related to York"
+lead: "Adam Barr"
+developers: ["Ben Silverman", "Harry Wickham"]
+link: "https://yordle.yordevs.com"
+---
+
+---
+
+# Yordle
+
+Wordle took the internet by storm over the first few months of 2022, whilst the idea behind the game was easy to understand, completing the a wordle provided a sense of accomplishment and pride you couldn't help but show of to your friends and family. It was a perfect mix for a viral game.
+
+Here at Yordevs we not above jumping on a band wagon, and as multiple wordle clones started putting their own new twist on the game, we wanted to get involved. It didn't take much imaginiation to decide that our spin on wordle was going to be making all the answers York related... oh and replacing the squares with ducks when you share your results.
+
+Yordle uses the basic create-react-app framework, we decided we didn't need to use any addional frameworks on top of this as the site was going to be a single page application. We decided that we didn't want to store the answer in the users' browser before they had figured it out, so all answer calls are made to a remote heroku server which responds with the result of the guess, the answer is only sent after the 6th guess.
```

Comment:
Yordle uses the basic create-react-app framework, we decided we didn't need to use any **_additional_** frameworks on top of this as the site was going to be a single page application. We decided that we didn't want to store the answer in the users' browser before they had figured it out, so all answer calls are made to a remote heroku server which responds with the result of the guess, the answer is only sent after the 6th guess.